### ISSUE TITLE
Packaging `runnc`

### ISSUE THREAD

**Issue by lumjjb** [nabla-containers/runnc#49]
There should be a mechanism to install `runnc` to a system without needing to have a build environment. This is useful in the case of installing a runtime onto a worker for testing, which would not have a dev environment set up.

My thoughts on this is to be able to have a debian package so that dependencies can also be handled appropriately. As talked about in https://github.com/nabla-containers/runnc/issues/48.

**Comment by InTheCloudDan** [comment]
Are you looking to have Travis handle the packaging? I found this: https://travis.debian.net/ but was wondering if you're looking for something more independent to handle it.

**Comment by lumjjb** [comment]
Yup! - we would want to have it handled by travis or some sort of CI. :). 

In addition, we probably will have an additional make command for `make dpkg` or something too.
